packages/local_auth/local_auth_android/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -90,9 +92,27 @@ interface AuthCompletionHandler {
             .setConfirmationRequired((Boolean) call.argument("sensitiveTransaction"))
             .setConfirmationRequired((Boolean) call.argument("sensitiveTransaction"));
 
-    if (allowCredentials) {
-      promptBuilder.setDeviceCredentialAllowed(true);
```

Comment:
[This method](https://developer.android.com/reference/androidx/biometric/BiometricPrompt.PromptInfo.Builder#setDeviceCredentialAllowed(boolean)) was deprecated in favor of `setAllowedAuthenticators` ([documentation](https://developer.android.com/reference/androidx/biometric/BiometricPrompt.PromptInfo.Builder#setAllowedAuthenticators(int)), so all authentication methods need to be known here versus just a boolean indicating whether or not to allow device credentials. See changes below.